PlayKitBroadpeak/Classes/BroadpeakMediaEntryInterceptor.swift

```diff
@@ -128,6 +127,8 @@ extension BroadpeakMediaEntryInterceptor: PKMediaEntryInterceptor {
                         self?.messageBus?.post(BroadpeakEvent.Error(error: BroadpeakPluginError.smartLibError(Int(result.getErrorCode()), result.getErrorMessage())))
                     } else {
                         if let url = URL(string: result.getURL()) {
+                            //send SourceUrlSwitched event
+                            self?.messageBus?.post(InterceptorEvent.SourceUrlSwitched(originalUrl: contentURL.absoluteString,                                                               updatedUrl: url.absoluteString))
```

Comment:
Something weird in this line. It seems you missed `updatedUrl` parameter

Reply:
spaces fixing it